### DISCUSSION TITLE
Please help to change the upload directory!!!

### DISCUSSION THREAD

**Discussion by kokobrise** [immich-app/immich#5334] [Q&A] [answered]
Hello. I installed Immich on my Mac Mini M1 Ventura OS with Docker Desktop. I used the installation guide with Docker Compose [Recommended] from the site. I've never worked with Linux and Docker, but I'm technically literate, even if I sound like an idiot. I didn't change anything on the two files (.yml and .env) The program worked without problems. I am trying to change the upload directory and want it to be on the external drive I have mounted. The name is Dimo Disk, and the folder I want to access is Photos. Please tell me which of the two files should I change, where should I change it and what should be the path to get there.

**Comment by alextran1502** [comment, marked as answer]
1. Move the files in your previous UPLOAD_LOCATION to the new directory
2. Update `.env` UPLOAD_LOCATION with the new directory path

**Reply by kokobrise** [reply]
Thank you! It's work!

**Reply by arLevi** [reply]
that is causing immich to fail with an error: 
```
Failed to read upload/upload/.immich: Error: ENOENT: no such file or directory, open 'upload/upload/.immich'. AND The "upload" folder appears to be offline/missing, please make sure the volume is mounted with the correct permissions.
```

If i change back the `UPLOAD_LOCATION` to what it was previously , then it works fine
how can i solve it ? i really need to increase the diskspace in another directory

**Reply by bo0tzz** [reply]
You need to move the files over.

**Reply by arLevi** [reply]
actually i did, the issue was that the docker desktop didn't even saw my USB drive ... noticed it only during backup/restore that i'm unable to mount it. the errors i received from docker were misleading..
